test/EVM/Test/Tracing.hs

```diff
@@ -289,12 +285,13 @@ evmSetup contr txData gaslimitExec = (txn, evmEnv, contrAlloc, fromAddress, toAd
     evmEnv = EVMToolEnv { coinbase   =  0xff
                         , timestamp   =  Lit 0x3e8
                         , number      =  0x0
-                        , prevRandao  =  0x0
                         , gasLimit    =  unsafeInto gaslimitExec
                         , baseFee     =  0x0
                         , maxCodeSize =  0xfffff
                         , schedule    =  feeSchedule
                         , blockHashes =  blockHashesDefault
+                        , withdrawals =  mempty
+                        , currentRandom =  42
```

Comment:
extra space

Reply:
Funnily enough everything has extra space above ;) It's all 2 spaces. But I now adjusted everything to be 1 space and to be aligned!